roles/iam-user/tasks/create_users.yml

```diff
@@ -0,0 +1,26 @@
+
+---
+- name: Create IAM users
+  iam:
+    state: present
+    iam_type: user
+    name: "{{ item.name }}"
+    groups: "{{ item.groups }}"
```

Comment:
I could use the same structure to create the group and policy from expected
iam_policy_[group].json file ?